### PR TITLE
Pip freeze all dependencies

### PR DESCRIPTION
**Pull request by maaktweluit**
Deja-vu, sorry i know!

This time lets merge this before anyone changes requirements :)

### CODE REVIEW

**Comment by etam** [comment]
I think it's OK. What about requirements-lint.txt and requirements-test.txt?

File: requirements.txt [hunk 1]
```diff
@@ -1,59 +1,115 @@
...
-cbor2==3.0.4
-base58
-ndg-httpsclient
-certifi
-tabulate
-PyYAML>=3.12
-crossbar==17.12.1
 txaio==2.8.2
-pydispatcher
-Crypto
```
Comment (etam) [inline requirements.txt:37]: What happened with `Crypto`? Build failed, because it's missing.

**Review by etam — CHANGES_REQUESTED** [review]

**Comment by maaktweluit** [comment]
@etam i skipped the -test and -lint since they are not part of the build, do you think they should be frozen too?

File: requirements.txt (hunk 1 above)
Comment (maaktweluit) [reply requirements.txt:37]: its not in pypi as `Crypto`... remember this was a problem last time too, will check that branch, i fixed it there :)

**Comment by etam** [comment]
> i skipped the -test and -lint since they are not part of the build, do you think they should be frozen too?

I think yes. For exactly the same reason, we're freezing golem's requirements.

**Comment by maaktweluit** [comment]
@etam done, builds also seem fixed :D

**Comment by etam** [comment]
`Double requirement given: six==1.11.0 (from -r requirements-test.txt (line 24)) (already in six==1.10.0 (from -r requirements.txt (line 99)), name='six')`

**Comment by maaktweluit** [comment]
six and pyelleptic fixed in the new commit

**Comment by maaktweluit** [comment]
@etam please check again, all errors are solved from my side :D

**Review by etam — APPROVED** [review]
frozen

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**